File: code_blocks/integrations/third-party-integrations/braze_1.swift
import SwiftUI
import RevenueCat
import Appboy_iOS_SDK

@main
struct MyApp: App {
    init() {
        Purchases.configure(withAPIKey: "public_sdk_key", appUserID: "my_app_user_id")
        Appboy.sharedInstance()?.changeUser("my_app_user_id")
        Purchases.shared.attribution.setAttributes([
            "$brazeAliasName": "name",
            "$brazeAliasLabel": "label"
        ])
    }

    var body: some Scene {
        WindowGroup {
            ContentView()
        }
    }
}

File: code_blocks/integrations/third-party-integrations/braze_1.dart
import 'package:purchases_flutter/purchases_flutter.dart';
import 'package:braze_plugin/braze_plugin.dart';

Future<void> init() async {
  await Purchases.configure(
    PurchasesConfiguration('public_sdk_key')..appUserID = 'my_app_user_id',
  );
  BrazePlugin.changeUser('my_app_user_id');
  await Purchases.setAttributes({
    '\$brazeAliasName': 'name',
    '\$brazeAliasLabel': 'label',
  });
}
